contents/blog/how-to-treat-your-pricing-like-a-product.md

```diff
@@ -0,0 +1,102 @@
+
+This is bad. If pricing is so important to a business that it somehow directly or indirectly determines nearly everything else, we simply cannot be okay with it being dead. 
+
+And to keep it alive, we’ve got to treat it like a product.
+
+But how? I’m glad you asked.
+
+## How to treat your pricing like a product
+
+### 1. Hire or assign a team to own pricing
+A product without an owner is as good as dead. Put someone – or a team of someones – in charge of your pricing and make it a significant portion of their responsibility. If the owner doesn’t have enough bandwidth to work on it, we’re essentially without an owner.
+
+At PostHog, the Growth team (hi! 👋) owns pricing, and I’d estimate it encompasses more than half of our overall responsibility.
+
+### 2. Understand who your customers are and their needs
+Every product has customers, and your pricing product is no different. Who is the customer will vary by business, but it essentially comes down to one question: which people need the pricing (and its tooling) to serve their specific needs?
+
+For PostHog, our pricing product’s customers are:
+
+- **Our users (external)**, who need to clearly see and understand our pricing model so they can decide if they want to sign up and subscribe.
+
+- **Other product teams (internal)**, who need a system that makes it easy and reliable for them to track usage, gate features, subscribe new customers, upsell, etc.
+
+- **Customer success (internal)**, who need a system that allows them to make custom packages for enterprise customers, anticipate invoices, manage accounts, and more.
+
+It’s important to also determine how each customer will transfer value to your product. Since people aren’t paying for pricing as a product (unless, of course, the product you sell is pricing as a product) then the value transfer is murkier. It often comes down to usage, or even revenue on the other products. Whatever it is, understand and track it so you can gauge the usefulness of your efforts.
+
+### 3. Build your tooling to satisfy the customers’ needs
+
+Depending on the complexity of your business, and the other products you sell, the tooling might be simple or rather elaborate. Sometimes just a basic Stripe integration will do; other times, like at PostHog, it’s a large service that runs all sorts of important business logic to keep it all tracking properly, in sync, and up-to-date. 
+
+Of course, based on my definition above, this tooling needs to be treated like a proper product from the engineers themselves. It needs to be tested, it needs to be reliable, and it needs to have hands on it all the time. Code without testing or maintenance is, well, dead. And products that are dead are not products.
+
+### 4. Build/integrate the tooling for experimenting
+
+Nearly any good product built today wasn’t just built once and assumed to be the best possible version. There are always things to try. For a pricing product this can be experimenting with things such as:
```

Comment:
```suggestion
Nearly any good software product today wasn’t just built once and assumed to be the best possible version. There are always things to try. For a pricing product, this can be experimenting with things such as:
```